benches/main.rs

```diff
@@ -0,0 +1,272 @@
+#![cfg(feature = "benches")]
```

Comment:
Would it be possible to use a hand-rolled `harness = false` scheme or something like `criterion` that works on stable?